doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+******************
+Basic Installation
+******************
+
+The following will show how to install both SmartSim and SmartRedis.
+
+.. note::
+  For users on platforms with a 'site install' of SmartSim please follow
+  :ref:`these instructions <site install>`.
```

Comment:
When I select this link it takes me to `doc/installation_instructions_basic.rst`, the same doc, which I think should not happen?

Reply:
I also am not sure that its taking the user to the right page, it just moves to the top of the doc

Reply:
negative